v5_enum.go

```diff
@@ -1,5 +1,13 @@
 package bybit
 
+// AccountType :
+type AccountType string
+
+const (
+	UnifiedAccount AccountType = "UNIFIED"
+	NormalAccount  AccountType = "CONTRACT"
+)
```

Comment:
nits:

I want to make it easy to predict from the type name.

```suggestion
const (
	AccountTypeUnified AccountType = "UNIFIED"
	AccountTypeNormal  AccountType = "CONTRACT"
)
```